docs/examples.rst

```diff
@@ -666,3 +666,14 @@ Finally, you can exclude single attributes from certain methods:
    ...     password = attr.ib(repr=False)
    >>> C("me", "s3kr3t")
    C(user='me')
+
+Alternatively, to influence how the generated ``__repr__()`` method formats a specific attribute, specify a custom callable to be used instead of the `repr()` built-in function:
```

Comment:
`repr()` is using markdown

Reply:
fixed in e6c98fc52102989a84006ea41f6d87181cfd920d

Reply:
(FWIW this syntax *does* work in rst if you set `default_role`, which is totally great).